test/SpokePoolClient.fills.ts

```diff
@@ -53,91 +45,54 @@ describe("SpokePoolClient: Fills", function () {
     await setupTokensForWallet(spokePool, relayer1, [erc20, destErc20], weth, 10);
     await setupTokensForWallet(spokePool, relayer2, [erc20, destErc20], weth, 10);
 
-    quoteTimestamp = Number(await spokePool.getCurrentTime());
-    fillDeadline = quoteTimestamp + 600;
-  });
-
-  it("Correctly fetches fill data single fill, single chain", async function () {
-    const deposit: Deposit = {
+    const spokePoolTime = Number(await spokePool.getCurrentTime());
```

Comment:
OOC why did we change this from `quoteTimestamp`?

Reply:
The value is the same but the name changed to `spokePoolTime` because we are setting multiple inputs in the Deposit() based on it (quoteTimestamp, fillDeadline).